core/pfe-core/controllers/toggle-controller.ts

```diff
@@ -145,7 +145,7 @@ export class ToggleController implements ReactiveController {
     };
   }
 
-  constructor(public host: ReactiveControllerHost, options?: ToggleControllerOptions) {
+  constructor(public host: ReactiveControllerHost & HTMLElement, options?: ToggleControllerOptions) {
```

Comment:
Please revert. This shouldn't require HTML element, since there can be cases where the host is not an element, for example framework shims

See https://github.com/lit/lit/blob/main/packages/react/src/use-controller.ts

A little extra consideration here and now will save us much headache in the near future. 

Non element hosts can take a getter function in the options initializer